src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -95,6 +95,29 @@ def register_content_settings_argument(self, settings_class, update, arg_group=N
                        'properties listed below is set, then that property will be cleared.',
                        arg_type=get_three_state_flag())
 
+    def register_content_settings_argument_track2(self, settings_class, update, arg_group=None, guess_from_file=None):
```

Comment:
Can we add an argument like `include_content_md5` to original func `register_content_settings_argument()` to reduce duplicate code?